modules/local/paraclu.nf

```diff
@@ -0,0 +1,49 @@
+def VERSION = '10' // Version information not provided by tool on CLI
+
+process PARACLU {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::paraclu=10" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/paraclu%3A10--h9a82719_1' :
+        'quay.io/biocontainers/paraclu:10--h9a82719_1' }"
+
+    input:
+    path(bed) // path: of pooled ctss file
+    val(min_cluster) // integer: minimum cluster size
+
+    output:
+    path("*.bed"), emit: bed
+    path "versions.yml", emit: versions
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    #  "BED into minus and positive"
+    sed '/+/d' $bed > all_neg
+    grep + $bed > all_pos
+
+    #  "CTSS output to paraclu format"
+    awk -F "\t" '{print\$1"\t"\$6"\t"\$2"\t"\$5}' < all_pos > ctss_pos_4P
```

Comment:
@drpatelh - could you comment here? There are a lot of these modules with many operations grouped together either in the module itself or in a bash script in `bin`.